.github/workflows/ci.yml

```diff
@@ -19,13 +19,11 @@ jobs:
           submodules: true
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@dd05243424bd5c0e585e4b55eb2d7615cdd32f1f
-        with:
-          workspaces: starknet-foundry
       - uses: software-mansion/setup-scarb@v1.0.1
         with:
           scarb-version: ${{ env.SCARB_VERSION }}
-      - run: cargo test --lib -p forge
-      - run: cargo test integration -p forge
+      - run: cargo test --release --lib -p forge
+      - run: cargo test --release integration -p forge
```

Comment:
What does the `release` flag do in this context?
Does it speed up the test execution in exchange for compile time?
Also, does it not disable some assertion checks?

Reply:
> Does it speed up the test execution in exchange for compile time?
Yes

The test behaviour doesn't change

Reply:
What about this?
https://stackoverflow.com/a/29822963